bedrock/firefox/templates/firefox/welcome/page2.html

```diff
@@ -0,0 +1,65 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/welcome/page2" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{{ _('Pocket - Save news, videos, stories and more') }}{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
+
+{% block body_class %}welcome-page2{% endblock %}
+
+{% block content_intro %}
+  <h1 class="c-page-title">{{ _('Your time online is worth protecting') }}</h1>
+  {# L10n: "Pocket" is a product name and shouldn't be translated. #}
+  <p class="c-page-tagline">{{ _('Discover and save stories in Pocket — and come back to them when you’re free.') }}</p>
```

Comment:
- I think that we should consider making `page-intro` into a `mzp-c-hero`.
<img width="1254" alt="Screen Shot 2019-10-15 at 12 34 50 PM" src="https://user-images.githubusercontent.com/30483988/66863944-f34b6880-ef48-11e9-80db-727ec10d2b8f.png">

- I also think we should strongly consider adding the "Activate Pocket" button to the hero section. Especially considering that this page is very similar in function to the whatsnew page.

- We should also write some tests for the new pocket button. :+1:

Reply:
I tried changing it to the hero but then I had to customize it because the design doesn't use the standard component (different sizes, padding, etc). So I'm not sure which is worse: a customized component that can break if the dependency changes upstream, or another non-standard, non-component. Feels like debt either way.

The button isn't part of the hero because the design didn't put it there. This one is following the pattern established on the first page so if we move the button we should do it on both. I'm reluctant to make that kind of design change at this stage though since these have already been through so much review. Seems like a good thing we could test later though.

I'm working on adding a test for the button at least.

Reply:
Thanks Craig! That's all fair enough. Considering that this is a set of pages, in the future we should push to make these templates modular in protocol like the whatsnew page. (is preaching to choir, haha)